frame/generic-asset/src/lib.rs

```diff
@@ -0,0 +1,1369 @@
+// Copyright 2019-2020
```

Comment:
this file shouldn't have been committed, looks like a bad merge

Reply:
FML

TY